SeleniumWebDriverBasics/SeleniumWebDriverBasics/Browser.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+
+namespace SeleniumWebDriverBasics
+{
+    public class Browser
+    {
+        public static IWebDriver driver;
+        private static Browser currentInstance;
+        public static BrowserFactory.BrowserType currentBrowser;
+        public static int ImplWait;
+        public static double timeOutForElement;
+        public static string browser;
+
+        private Browser()
+        {
+            InitParamas();
+            driver = BrowserFactory.GetDriver(currentBrowser, ImplWait);
+        }
+
+        public static void InitParamas() 
+        {
+            ImplWait = Convert.ToInt32(Configuration.ElementTimeout);
+            timeOutForElement = Convert.ToDouble(Configuration.ElementTimeout);
+            browser = Configuration.Browser;
+            Enum.TryParse(browser, out currentBrowser );
+        }
```

Comment:
Conversion and parsing can be made in the Configuration straightly, you don't need the InitParamas() method then.